src/main/java/org/snomed/snowstorm/core/data/services/ConceptUpdateHelper.java

```diff
@@ -499,7 +499,9 @@ private <T> List<ReferenceSetMember> updateMetadataRefset(Map<String, Set<String
 							.noneMatch(id -> id.equals(member.getMemberId()))) {
 
 				member.setActive(false);
-				member.markChanged();
+				if (!member.buildReleaseHash().equals(member.getReleaseHash())) {
+					member.markChanged();
+				}
```

Comment:
In an earlier conversation, @kaicode does not think this is the correct solution as we shouldn't care about the release hash.

Reply:
@dmcgihtsdo I agree with Kai's comment. I think we could do some refactoring in this method.

Reply:
@dmcgihtsdo  I will refactor this and we can catch up tomorrow.